apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -116,46 +120,84 @@ class MenuBubbleUnwrapped extends Component {
       <div style={style} className={className}>
         {this.renderMenuItems()}
         {/* These elements are used to draw the 'tail' with CSS */}
-        <span style={tailBorderStyle}/>
-        <span style={tailFillStyle}/>
+        {!this.props.minimalStyle &&
+          <span style={tailBorderStyle}/>
+        }
+        {!this.props.minimalStyle &&
+          <span style={tailFillStyle}/>
+        }
       </div>
     );
   }
 }
 export const MenuBubble = Radium(MenuBubbleUnwrapped);
 
+class MenuBreak extends Component {
+
+  render() {
+    const style = {
+      borderTop: '1px solid ' + color.lighter_gray,
+      marginTop: STANDARD_PADDING/2,
+      marginBottom: STANDARD_PADDING/2,
+      marginLeft: STANDARD_PADDING,
+      marginRight: STANDARD_PADDING,
+    };
+    return (
+      <div style={style}></div>
+    );
+  }
+}
+
+PopUpMenu.MenuBreak = Radium(MenuBreak);
```

Comment:
Does this component get any benefit from Radium?  I don't see any style arrays, hover behaviors, or styles that need browser-prefixing.

If not, and especially because this has no props, this would make a great functional stateless component.

Reply:
There is one ":hover" in here (line 177) so I would be hesitant to remove Radium.

Reply:
But that's on the `PopUpMenu.Item` component, not on the `PopUpMenu.MenuBreak` component.